Essentials/src/messages_nl.properties

```diff
@@ -502,8 +502,8 @@ teleporting=\u00a76Bezig met teleporteren...
 teleportInvalidLocation=De waarde van co\u00f6rdinaten kan niet hoger dan 30000000 zijn
 teleportNewPlayerError=\u00a74Fout bij het teleporteren van nieuwe speler\!
 teleportRequest=\u00a7c{0}\u00a76 vraagt of hij naar u kan teleporteren.
-teleportRequestAllCancelled=\u00a76All outstanding teleport requests cancelled.
-teleportRequestSpecificCancelled=\u00a76Outstanding teleport request with {0} cancelled.
+teleportRequestAllCancelled=\u00a76Alle openstaaande teleport verzoeken zijn geannuleerd.
+teleportRequestSpecificCancelled=\u00a76Alle openstaande teleport verzoeken van {0} geannuleerd.
```

Comment:
I don't think this should contain `Alle`? It's for **one** specific teleport request, unlike the message above which is for all of them. (Correct me if I'm wrong and this actually makes more sense in Dutch.)

Reply:
Uhm.... That is an typo, it should say

teleportRequestSpecificCancelled=\u00a76Openstaande teleport verzoek van {0} geannuleerd.

Reply:
@DutchUser Is there any chance you could fix that message? Once done, I can merge this.